css/styles.css

```diff
@@ -709,3 +710,458 @@ a {
   color: #172b4d;
   transform: 0.8s;
 }
+
+.nav-link .navigation-bar {
+  display: none;
+}
+
+/** Desktop Media css styles **/
+@media only screen and (min-width: 768px) {
+  .root-container {
+    display: grid;
+    all: unset;
+    width: 1440px;
+    height: 5125px;
+    background: #e5e5e5;
+  }
+
+  .header-section {
+    background-image: url("../image/header-desktop-bg.svg");
+    background-repeat: no-repeat;
+    flex-wrap: wrap;
+    align-items: center;
+    border-bottom-left-radius: 120px;
+    width: 1440px;
+    height: 916px;
+    left: 0;
+    top: 0;
+    margin: 0 auto 0 auto;
+  }
+
+  .developer-name {
+    width: 380px;
+    height: 122px;
+    font-family: 'Raleway', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 48px;
+    display: flex;
+    margin-left: 100px;
+    margin-top: 150px;
+    align-items: center;
+    color: #172b4d;
+  }
+
+  .header-text {
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    width: 920px;
+    font-family: 'Poppins', sans-serif;
+    text-align: justify;
+    color: #344563;
+  }
+
+  .link-image {
+    display: flex;
+    justify-content: flex-start;
+  }
+
+  .connect-link {
+    width: 40%;
+    font-weight: 500;
+    font-size: 16px;
+    font-style: normal;
+    line-height: 24px;
+    color: #7f8cff;
+    font-family: 'Poppins', sans-serif;
+    order: 0;
+    flex: 0 0 auto;
+  }
+
+  .connect-icon {
+    list-style-type: none;
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-start;
+    width: 170px;
+    height: 20px;
+    margin-top: auto;
+  }
+
+  .nav-link {
+    all: unset;
+    position: sticky;
+    z-index: 1;
+    top: 0;
+    left: 0;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 0.2rem 3.8rem;
+    list-style: none;
+  }
+
+  .nav-link .navigation-bar {
+    all: unset;
+    display: flex;
+    gap: 0.2rem;
+  }
+
+  .navigation-bar a {
+    color: #344563;
+  }
+
+  .nav-link img {
+    display: none;
+  }
+
+  .tonic-card {
+    display: flex;
+    flex-direction: row;
+    width: 1156px;
+    height: 496px;
+    background: #fff;
+    border-radius: 24px;
+    border: 1px solid #dfe1e6;
+    margin: 50px 0 0 100px;
+  }

```

Comment:
- [ ] According to the figma design the project cards have alternating positions of the text and images, the first card has the image to the left and text to the right while the second card has an image to the right and the text to the left. Your project has all text to the right and the images to the left, kindly fix that. Cheers 👍 




Figma Design             |  Your Design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/60618877/179267899-d79f2329-9be8-4b6c-ae60-92422a719b80.png)  |  ![image](https://user-images.githubusercontent.com/60618877/179268199-478d4299-7fec-45b2-be56-0c022eef6a11.png)

Reply:
@sumairq Thank you for the detailed review. Now regarding the card alternating position, you may please remember I'm following my initial mobile page design, which I'm now making responsive for desktop in this part of the task. Kindly clarify more on this review.